architecture.md

```diff
@@ -34,13 +27,7 @@ The entry point of the container is the Go UAST normalizer, which communicates
 internally with the language-specific code parser. This is how the Python driver
 looks like:
 
-```mermaid
-graph LR
-    Server-- stdin/stdout<br/>gRPC -->UAST
-    subgraph Container
-        UAST["UAST Normalizer<br/>(Go)"]-- stdin/stdout<br/>JSON -->AST["Code Parser<br/>(Python)"]
-    end
-```
+![Driver Architecture](./images/architecture-driver.png)
```

Comment:
Maybe you should explain what the arrows mean in the caption of the image or on the text.